xsd/sdp-manifest.xsd

```diff
@@ -32,6 +32,15 @@
 					</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+            <xsd:element name="data" minOccurs="0" maxOccurs="1" type="DokumentData">
+                <xsd:annotation>
+                    <xsd:documentation>
+                        Ekstra data som beriker dokumentet i postkassen.
+                        Se https://begrep.difi.no/SikkerDigitalPost/utvidelser for en oversikt over hvilke utvidelser
```

Comment:
Jeg kjenner ikke spesielt godt til den tekniske løsningen bak begrepskatalogen, så Difi får vurdere hvilken URL-struktur som er hensiktsmessig for å dokumentere disse utvidelsene :)